.github/workflows/compare-versions/action.yml

```diff
@@ -0,0 +1,53 @@
+name: 'Compare Versions'
```

Comment:
I would actually think it need to be even more simple than using the env-param here, instead of passing.
I think its better to have a flag in the json file saying valkey or not, and simply extracting this flag and using in an if statement instead of so much logic for a simple thing.

Reply:
Decided to remove all specific valkey checks. If installed and linked correctly redis-server should be with the required version.